### PR TITLE
Provide prevention measures

### PR DESCRIPTION
**Pull request by HiimHyy**
Awesome material!
Trying to contribute more to site.

### CODE REVIEW

**Review by davidchiii — DISMISSED** [review]
great addition! Thanks for contributing

File: docs/web-exploitation/sql-injection/what-is-sql-injection.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ SELECT * FROM users WHERE username='' OR true
 
 This will return every row in the table because each row that exists must be true.
 
-We can also inject comments and termination characters like `--` or `/*` or `;`. This allows you to terminate SQL queries after your injected statements. For example `'-- ` is a common SQL injection payload.
+We can also inject comments and termination characters like `--` or `/*` or `;`. This allows you to terminate SQL queries after your injected statements. For example `'--` is a common SQL injection payload.
```
Comment (M0rtale) [inline docs/web-exploitation/sql-injection/what-is-sql-injection.md:41]: block: SQL requires at least 1 space after double dash for a valid comment

https://dev.mysql.com/doc/refman/8.0/en/comments.html#:~:text=In%20MySQL%2C%20the%20%2D%2D%20(double,as%20discussed%20in%20Section%201.6.

**Review by M0rtale — CHANGES_REQUESTED** [review]
✨

**Review by HiimHyy — COMMENTED** [review]
resolve sql injection example

File: docs/web-exploitation/sql-injection/what-is-sql-injection.md (hunk 1 above)
Comment (HiimHyy) [reply docs/web-exploitation/sql-injection/what-is-sql-injection.md]: fixed in new commit

**Review by davidchiii — APPROVED** [review]
fixed. thanks @M0rtale :)

**Review by M0rtale — APPROVED** [review]
LGTM

**Pull request merged**